fastlane_core/lib/fastlane_core/analytics/analytics_session.rb

```diff
@@ -191,7 +191,7 @@ def operating_system_version
       os = self.operating_system
       case os
       when "macOS"
-        return `SW_VERS -productVersion`.strip
+        system('sw_vers') ? return `sw_vers -productVersion`.strip : ''
```

Comment:
Should we use a string like `unknown` like we do with unknown p-hashes?  (easier to query against, maybe?)